Makefile

```diff
@@ -123,3 +123,11 @@ clone-samples:
 		tar -xJvf samples/linux/clean/$$file -C samples/linux/clean; \
 	done
 	tar -xJvf samples/macOS/clean/bincapz.xz -C samples/macOS/clean
+
+.PHONY: install-crane
+install-crane:
+	go install github.com/google/go-containerregistry/cmd/crane@latest
```

Comment:
make shouldn't modify files outside of this directory. I suggest setting GOBIN and pinning this to a specific version for consistent output. This will also allow you to cache the binary without reinstalling it every run. Example:

https://github.com/chainguard-dev/osquery-defense-kit/blob/df577d4f1c08053a4a3bc53a5461736662fb3c82/Makefile#L6

Reply:
Makes sense 👍🏻 

Updated in [`d220c8f` (#438)](https://github.com/chainguard-dev/bincapz/pull/438/commits/d220c8fe0e66e1c1e5fd5085d3868900c3622f2a).